lib/carrierwave/mounter.rb

```diff
@@ -38,7 +38,7 @@ def uploaders
     end
 
     def cache(new_files)
-      return if not new_files or new_files == ""
+      return if !new_files || new_files.blank?
```

Comment:
`!new_files` is unnecessary, because `nil.blank?` is true.

Reply:
I overlooked it.
Thank you.